Source/DataSources/StaticGeometryPerMaterialBatch.js

```diff
@@ -130,7 +130,7 @@ import Property from './Property.js';
                     geometryInstances : geometries,
                     appearance : new this.appearanceType({
                         material : this.material,
-                        translucent : this.material.isTranslucent(),
+                        // omitting translucent property to override it
```

Comment:
Assuming we do take this change, a link back to this PR for details and/or a more expanding explanation would be helpful